cms/templates/widgets/user_dropdown.html

```diff
@@ -5,6 +5,9 @@
   from django.urls import reverse
   from django.utils.translation import ugettext as _
   from student.roles import GlobalStaff
+  from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
+
+  lms_logout_url = configuration_helpers.get_value('LMS_BASE') + reverse('colaraz_features:auth_logout_redirect')
```

Comment:
Use `format` for string concatenation ?

Reply:
does it make any difference?

Reply:
depends if consistency is required or not

Reply:
basically, according to my understanding, format() is not solely for string concentrations. It is used when we have some hardcoded strings in it, but in this scenario its just two variables.